pkg/manifests/manifests.go

```diff
@@ -1281,6 +1281,10 @@ func (f *Factory) PrometheusK8s(host string, grpcTLS *v1.Secret, trustedCABundle
 		}
 	}
 
+	if f.config.ClusterMonitoringConfiguration.PrometheusK8sConfig.QueryLogFile != "" {
```

Comment:
you might find this `go idiom` useful
```go
if logFile := f.config....QueryLogFile; logFile != "" {
  p.spec.QueryLogFile =  logFile
}
```

Reply:
we could indeed copy correctly. If you look further up this function though you will see similar pattern https://github.com/openshift/cluster-monitoring-operator/pull/1373/files/4a3d1247b17a8fc816096224041213bf351d8adf#diff-1f9d2d18d4cabd8262e585b932330183f7cb9b544ac9f27c8bcb3ef9c8d14376R1251-R1257

Often I find for readability it is best to match the existing style, wdyt?

Reply:
I agree, it is better to follow the style in general but it in particular instance, I am not quite convinced that it was intended as a convention, rather I feel it is something  the original author may have missed ...  but again, this is only a `nit`, so please feel free to ignore :-)

Reply:
no its a valid point, and I agree that we should make the code idiomatic. I'm just not sure this PR is the right place to make those changes. Perhaps we should curate a list of things we need to "tidy" in the codebase? I've done a similar cleanup on the e2e tests already.

Reply:
Yes, makes sense to move the cleanup to a different PR 🤗

Reply:
Late to the party but technically we don't need the `if` check since an empty value disables query logging.

https://github.com/prometheus-operator/prometheus-operator/blob/2a802a189b1230ef04e82c3e5ecad9de80668db3/pkg/prometheus/promcfg.go#L264-L268

https://github.com/prometheus/prometheus/blob/1270b87970baeb926fcce64552db5c744ffaf83f/cmd/prometheus/main.go#L558-L561